model/verification/chunkDataPackRequestStatus.go

```diff
@@ -0,0 +1,43 @@
+package verification
+
+import (
+	"time"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/model/flow/filter"
+)
+
+// ChunkRequestStatus is a data struct represents the current status of fetching
+// chunk data pack for the chunk.
+type ChunkRequestStatus struct {
+	*ChunkDataPackRequest
+	Targets     flow.IdentityList
```

Comment:
Note: for now, we keep the target IDs in addition to the agrees and disagrees sets. The target ids is the universal set that agrees and disagrees belong to. The reason is `Targets` is an `IdentityList` but the `Agrees` and `Disagrees` are `IdentifierList`s. The built-in `Sample` feature is only available for the `IdentityList`, which we are using down there 👇  to sample the target execution nodes for dispatching chunk data requests. 

However, if the `IdentifierList` would have such a built-in sample functionality, we would go ahead with that. So, I created this issue to add a `Sample` method to the `IdentifierList` and then refactor this part of code (i.e., `ChunkRequestStatus` to only involve the `Agrees`, and discard the `Targets`.